casadm/cas_lib.c

```diff
@@ -3041,6 +3041,8 @@ int standby_activate(int cache_id, const char *cache_device)
 
 	if (set_device_path(cmd.cache_path, sizeof(cmd.cache_path),
```

Comment:
I would move this message inside set_device_path() - it is now printed almost everywhere whenever set_device_path() is printed + there are cases where set_device_path() prints other messages and it would be good to have just one.